letsdrill.rb

```diff
@@ -1,24 +1,40 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  grade = case integer
+    when 90..100 then "A"
+    when 80..89 then "B"
+    when 70..79 then "C"
+    when 60..69 then "D"
+    when 0..59 then "F"
+    else "Invalid Score"
+  end
+
+  return grade
```

Comment:
Returns are implied in Ruby, try running this line without it. In fact you can try and pass this line without this line at all!